scout/server/blueprints/variant/templates/variant/buttons.html

```diff
@@ -146,7 +150,7 @@
       {% if case.upd_sites_beds %}
         <input type="hidden" name="upd_sites_bed" value="{{case.upd_sites_beds|join(',')}}">
       {% endif %}
-      <button class="btn btn-outline-secondary btn-sm" name="align" value="bam" type="submit">IGV viewer</button>
+      <button class="btn btn-secondary btn-sm" name="align" value="bam" type="submit">IGV viewer</button>
```

Comment:
Probably drop the btn-sm here as well - it looks a little odd next to the RNA track button.

Reply:
![Screenshot 2021-04-08 at 11 32 33](https://user-images.githubusercontent.com/758570/114005537-e9fe3c00-985f-11eb-87be-b9bcf388517d.png)

Reply:
Fixed it like this:

![image](https://user-images.githubusercontent.com/28093618/114034263-4ff9bc00-987e-11eb-9137-c394189c3c3f.png)